simulator/simulator.h

```diff
@@ -77,6 +79,10 @@ class Simulator : public CPUModel
     {
         return create_functional_simulator( isa, false);
     }
+protected:
+    void set_isa( std::string_view i) { isa = i; }
+private:
+    std::string_view isa;
```

Comment:
For ownership, it's safer to use `std::string`.
`std::string_view` is just a pointer.